src/focus-ring.js

```diff
@@ -31,7 +31,7 @@ document.addEventListener('DOMContentLoaded', function() {
    */
   function focusTriggersKeyboardModality(el) {
     var type = el.type;
-    var tagName = el.tagName.toLowerCase();
+    var tagName = el.tagName;
 
     if (tagName == 'input' && inputTypesWhitelist[type] && !el.readonly)
```

Comment:
Be sure to test these changes. I’m not sure `'input'` will work because it should now be uppercase?